Assets/Python/RFCUtils.py

```diff
@@ -677,274 +569,180 @@ def isAVassal(self, iCiv):
 		return False
 
 	#Barbs, RiseAndFall
-	def squareSearch(self, tTopLeft, tBottomRight, function, argsList, tExceptions = () ): #by LOQ
+	def squareSearch(self, tTopLeft, tBottomRight, function, argsList, tExceptions = (), bOuter = False): #by LOQ

```

Comment:
Not necessary. The bOuter is not needed, but I  forgot to remove it.

The argslist isn't really a list of input arguments it seems. It is used as a list of civs. I think it's better to change the name to civList or something and give it an empty list as default value.